### ISSUE TITLE
The demo works in my phone and my tablet, but when i use the code example only works in my phone

### ISSUE THREAD

**Issue by javcastro3** [schmich/instascan#229]
Simply, I have been tested the library and the demo works perfectly in my phone and in my tablet using chrome browser in both, but when I use the code on my custom page, it doesnt work in the tablet but in the phone still working. 

Any help?

**Comment by T620** [comment]
I had the same problem. I then used `instascan.min.js` sourced from the demo app, which, in case you're wondering, is available here: https://rawgit.com/schmich/instascan-builds/master/instascan.min.js.
Perhaps the example in the repo is outdated?

**Comment by javcastro3** [comment]
Yes, that solves my problema for now and update chrome browser too, thank you. However, I will have to do more test in other tablets with other Android versions to more or less, ensure this fix. 

I think too that the example is outdated, because if you take a look to the main page of rawgit [https://rawgit.com](url) it says "RawGit has reached the end of its useful life....GitHub repositories that served content through RawGit within the last month will continue to be served until at least October of 2019. URLs for other repositories are no longer being served." so I'm using the jsDelivr option getting this link [https://cdn.jsdelivr.net/gh/schmich/instascan-builds@master/instascan.min.js.](url) instead [https://rawgit.com/schmich/instascan-builds/master/instascan.min.js.](url)

**Comment by T620** [comment]
Might be a good idea to self-host the file.

**Comment by idsudeep** [comment]
Do i need any SSL certificate for running this code in phone .. its working on chrome pc but when i try to run through mobile it won't work ... what is the issues im  using this in my college project  ... please help me  to figure out this problem  ... thanks in advance  |  project is live on dllll.herokuapp.com

**Comment by javcastro3** [comment]
You don't need any certificate. I've checked your live demo. Maybe you need to add
`<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    <script src="https://cdn.jsdelivr.net/gh/schmich/instascan-builds@master/instascan.min.js"></script>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.1.10/vue.min.js"></script>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/webrtc-adapter/3.3.3/adapter.min.js"></script>
    <script type="text/javascript" src="https://rawgit.com/schmich/instascan-builds/master/instascan.min.js"></script>`

If you're using ios, in ios 10 and lower I couldn't make it work. iOS 10 and lower are not compatible with webRTC and instascan use that resource. 
I could share my code with you if you need

**Comment by idsudeep** [comment]
> You don't need any certificate. I've checked your live demo. Maybe you need to add
> `<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script> <script src="https://cdn.jsdelivr.net/gh/schmich/instascan-builds@master/instascan.min.js"></script> <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.1.10/vue.min.js"></script> <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/webrtc-adapter/3.3.3/adapter.min.js"></script> <script type="text/javascript" src="https://rawgit.com/schmich/instascan-builds/master/instascan.min.js"></script>`
> 
> If you're using ios, in ios 10 and lower I couldn't make it work. iOS 10 and lower are not compatible with webRTC and instascan use that resource.
> I could share my code with you if you need
If you can send me 
Thank you for being an important part of my project |sudeep.numb@gmail.com|
